mmv1/third_party/tgc/tests/data/example_google_dataflow_job.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "name": "//dataflow.googleapis.com/v1b3/projects/{{.Provider.project}}/locations/placeholder-2EsS2bm9/jobs",
+        "asset_type": "dataflow.googleapis.com/Job",
+        "ancestry_path": "{{.Ancestry}}/project/{{.Provider.project}}",
+        "resource": {
+            "version": "v1b3",
+            "discovery_document_uri": "https://www.googleapis.com/discovery/v1/apis/dataflow/v1b3/rest",
```

Comment:
I think this is https://dataflow.googleapis.com/$discovery/rest?version=v1b3 in the CAI list assets response. Let us make this consistent with CAI list asset response.

Reply:
done, thanks